presets/4.3/tune/mouseFPV_FFVCycle_Incisor_PT5_tune_filters.txt

```diff
@@ -0,0 +1,242 @@
+#$ TITLE:  FPVCycle Incisor/Prototype 5 6s Tune | mouseFPV 
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: FPVCycle
+#$ AUTHOR: mouseFPV
+#$ PARSER: MARKED
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION: <img src="https://user-images.githubusercontent.com/19867640/174462482-28bdcfec-1c3a-43db-99d7-50688b92f050.svg" width="100px" style="margin-left: auto; margin-right: auto; display: block;"/>
+#$ DESCRIPTION:
+#$ DESCRIPTION: # Tune for FPVCycle Incisor/Prototype 5 
+#$ DESCRIPTION:
+#$ DESCRIPTION: ## About:
+#$ DESCRIPTION: * If tune is too spicy, lower the master by a click or bring in a motor limit
+#$ DESCRIPTION: * Recommended 48k PWM
+#$ DESCRIPTION: * Uses Multi-Dynamic notich by default. Options for RPM filtering which use Dshot. Using DShot on Unsupported ESC's is Dangerous. See Warning.
+#$ DESCRIPTION: * RPM Filtering is recommended
+#$ DESCRIPTION: 
+#$ DESCRIPTION: ## Options:
+#$ DESCRIPTION: ### **General Options:**
+#$ DESCRIPTION: * **Dynamic Idle:** Enables Dynamic Idle for Freestyle 5"
+#$ DESCRIPTION: * **Set 93% Motor Limit:** Simulates 1800-1900kv motors down closer to 1700kv
+#$ DESCRIPTION: * **Enable Battery Sag Compensation:** Self Explanatory (see tooltip). Land at 3.5v/Cell or it's a bad time.
+#$ DESCRIPTION: * **Heavy Action Camera:** Increases the Pids/Master Slider
+#$ DESCRIPTION:
+#$ DESCRIPTION: ### **Filters:**
+#$ DESCRIPTION: * **Include RPM Filter DShot600 (F7 & Up)**: Enables RPM filtering. ESCs must support bi-directional. For F7 or better, sets Dshot600, 8k pidloop recommended (set manually).
+#$ DESCRIPTION: * **Include RPM Filter DShot300 (F4)**: Enables RPM filtering. ESCs must support bi-directional. For F4, sets Dshot300, 4k pidloop recommended (set manually). 
+#$ DESCRIPTION:
+#$ DESCRIPTION: ## Build Specs This Was Created On:
+#$ DESCRIPTION: * **Frame:** FPVCycle Incisor/PT5
+#$ DESCRIPTION: * **Motors:** FPVCycle 25mm Emerald 1880kv w/ 93 % Motor Limit 6s
+#$ DESCRIPTION: * **FC:** TMotor F7 (MPU6000)
+#$ DESCRIPTION: * **ESC:** TMotor F55a (non f3)
+#$ DESCRIPTION: * **Action Cam:** GoPro Hero 5 Session
+#$ DESCRIPTION: * **AUW:** 660-670g
+#$ DESCRIPTION:
+#$ DESCRIPTION: ## Fly Like mouseFPV | Recommendations Outside of Tune:
+#$ DESCRIPTION: * Apply mouseFPV Freestyle Rates
+#$ DESCRIPTION: * Use 250hz radio link if possible
+#$ DESCRIPTION: * **Set Jitter Reduction (feedforward_jitter_factor) to 14**
+#$ DESCRIPTION: * Set Feedforward Boost (feedforward_boost) to 10
+#$ DESCRIPTION:
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/254
+#$ WARNING: If You Choose To Include Filters Please See The Following:
```

Comment:
If You Choose To Include **RPM** Filters Please See The Following:
...because some filters are included by default

Reply:
Will update